packages/jest/src/plugins/plugin.spec.ts

```diff
@@ -25,6 +26,8 @@ describe('@nx/jest/plugin', () => {
       configFiles: [],
     };
 
+    setWorkspaceRoot(tempFs.tempDir);
```

Comment:
nit: is this needed? AFAICT, we already do this in the `TempFs` constructor: https://github.com/nrwl/nx/blob/master/packages/nx/src/internal-testing-utils/temp-fs.ts#L30-L32.

Reply:
Good catch, this came from the throw 💩 at the wall and see what sticks debugging portion of development 😅.

I've confirmed its not needed, and removed.